types/lhr.d.ts

```diff
@@ -87,6 +87,10 @@ declare global {
         weight: number;
         /** Optional grouping within the category. Matches the key of a Result.Group. */
         group?: string;
+        /** Any audit IDs closely relevant to this one */
```

Comment:
(applies to all comments)

please make a pass and 1) capitalize first letter 2) end with period